internal/pkg/cli/job_logs.go

```diff
@@ -78,10 +78,14 @@ func newJobLogOpts(vars jobLogsVars) (*jobLogsOpts, error) {
 // Validate returns an error if the values provided by flags are invalid.
 func (o *jobLogsOpts) Validate() error {
 	if o.appName != "" {
-		_, err := o.configStore.GetApplication(o.appName)
-		if err != nil {
+		if _, err := o.configStore.GetApplication(o.appName); err != nil {
```

Comment:
Do we also need to validate `envName` flag then? `appName` and `envName` are used in `initLogsSvc` which are called by `svc logs` from within `Execute()`